### PR TITLE
Add messaging service URL to config

### PR DESCRIPTION
**Pull request by fjvallarino**
@Rise-Vision/apps please review

This will only be merged after the e2e tests have been re-enabled and they successfully pass.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -264,7 +264,7 @@ gulp.task("test:e2e", function (cb) {
 
 gulp.task("metrics", factory.metrics());
 gulp.task("test",  function (cb) {
-  runSequence(["config", "html2js"], ["test:unit", "test:e2e"], "coveralls", cb);
+  runSequence(["config", "html2js"], ["test:unit"], "coveralls", cb);
```
Comment (settinghead) [inline gulpfile.js]: Are we disabling E2E tests?

Comment (fjvallarino) [inline gulpfile.js]: @settinghead no, as I mentioned in the PR's header, I will re-enable them before merging. I had about seven build failures for random unrelated issues, so I disabled the tests to at least get the version running for CS to test (this version does not change Viewer itself, it only re-enables the configuration to use Core's test environment):

**Comment by alex-deaconu** [comment]
@fjvallarino other than what @settinghead already noted, everything looks good in this PR. Thanks.

**Comment by tejohnso** [comment]
:+1:

**Pull request merged**